### PR TITLE
adding a step Verifying that Critical Operations Use Secure Communica…

### PR DESCRIPTION
**Pull request by xep624**
I didn't find lots of resources about adding additional channels, so I wrote it on a very high level. Please let me know if you have any good resources applicable to this point, so I will extend it.

### CODE REVIEW

File: Document/0x05g-Testing-Network-Communication.md [hunk 1]
```diff
@@ -302,44 +302,37 @@ The SSL pinning process should be implemented as described on the static analysi
...
-[Describe the best practices that developers should follow to prevent this issue.]
+Ensure that critical operations require at least one additional channel to confirm user's action. Each channel must not be bypassed to execute a critical operation.
 
 #### References
 
 ##### OWASP Mobile Top 10 2014
 
-* MX - Title - Link
-* M3 - Insufficient Transport Layer Protection - https://www.owasp.org/index.php/Mobile_Top_10_2014-M3
+This step is out of Mobile Top 10 Risks and should be applied only to sensitive applications.
```
Comment (commjoen) [inline Document/0x05g-Testing-Network-Communication.md]: "out of" --> "not in"

File: Document/0x05g-Testing-Network-Communication.md [hunk 2]
```diff
@@ -302,44 +302,37 @@ The SSL pinning process should be implemented as described on the static analysi
...
-
-[Describe how to assess this given either the source code or installer package (APK/IPA/etc.), but without running the app. Tailor this to the general situation (e.g., in some situations, having the decompiled classes is just as good as having the original source, in others it might make a bigger difference). If required, include a subsection about how to test with or without the original sources.]
-
-[Use the &lt;sup&gt; tag to reference external sources, e.g. Meyer's recipe for tomato soup<sup>[1]</sup>.]
 
-##### With Source Code
+#### Static Analysis
 
-##### Without Source Code
+Static analysis is not applicable to this point.
```
Comment (commjoen) [inline Document/0x05g-Testing-Network-Communication.md]: Not sure about this: the topic might be to broad to address it all at once. I guess you can have multiple different scenarios that tell something about the strength of the additional channel. For this I guess you can for now say "static analysis TBD".

**Comment by commjoen** [comment]
I guess there are a few items you can think off and search for docs for each individual items:
- tokens (RSA token, yubikey, card+cardreader, etc.)
- push notification
- SMS
- email
- data from another website you had to visit/scan
- data from a physical letter or physical entry point (e.g.:data you can only get after signing something at the office of a bank)

**Comment by xep624** [comment]
@commjoen I didn't find good resources about implementing each of item you've mentioned (token, push notification, sms etc.) and I don't have experience in using it, so I've just mentioned a list of them, but in future each item should be extended with instruction how such correct implementation should look like.

**Comment by b-mueller** [comment]
@xep624 - thanks, I'm merging!

**Pull request merged**